scripts/patch.js

```diff
@@ -26,8 +26,9 @@ require('make-promises-safe') // exit on error
 const ipfs = IpfsHttpClient()
 const pathTo = (file) => path.join(__dirname, '..', file)
 
+const event = new Date()
 const DIST_DOMAIN = 'dist.ipfs.io'
-const MFS_DIR = `/${DIST_DOMAIN}`
+const MFS_DIR = `/${DIST_DOMAIN}` + '_' + event.toISOString()
```

Comment:
This means that people who run this will end up with one MFS reference per-build instead of one per dist website. Given that my understanding is we're pinning all the files we add anyway the only problem that could come up with this is just it polluting your MFS root space.

nit: WDYT about having some nesting to keep `ipfs files ls /` cleaner? An example might be `/${DIST_DOMAIN}/${DIST_DOMAIN}` + '_' + event.toISOString()`

Reply:
I actually want to pollute MFS root.  The is so I see that I have multiple dists and remember to remove those that I no longer need.  This is not for the purpose of archiving dists, but to avoid stepping on a previous one.  Once I am happy with a dist, I expect to remove previous ones.

@Stebalien suggested in the previous review, that I use a random string here. I decided that date was more useful to determine which was the most recent.

Reply:
Honestly, keeping old ones should be _fine_ as it shouldn't actually use any more data. It also makes it easier to delete them (delete the root).

But either way. I don't have strong opinions.

Reply:
this is just a local thing for the builder we can always change later if it becomes a problem with no significant consequences, so we can just leave it as is and figure it out.